src/handlers/swap_handlers.ts

```diff
@@ -146,9 +146,13 @@ const findTokenAddressOrThrowApiError = (address: string, field: string, chainId
 };
 
 const parseStringArrForERC20BridgeSources = (excludedSources: string[]): ERC20BridgeSource[] => {
+    // Need to compare value of the enum instead of the key, as values are used by asset-swapper
+    // CurveUsdcDaiUsdt = 'Curve_USDC_DAI_USDT' is excludedSources=Curve_USDC_DAI_USDT
     return excludedSources
         .map(source => (source === '0x' ? 'Native' : source))
```

Comment:
Didn't we change this in swapper too?

Reply:
Asset-swapper does/should always use the value `Curve_USDC_DAI_USDT`